wt-status.c

```diff
@@ -1003,16 +1003,17 @@ static void wt_longstatus_print_other(struct wt_status *s,
 
 size_t wt_status_locate_end(const char *s, size_t len)
 {
-	const char *p;
-	struct strbuf pattern = STRBUF_INIT;
+        const char *p;
+        struct strbuf pattern = STRBUF_INIT;
```

Comment:
Why the indentation change?

Reply:
No reason I would like to admit publicly. `wt_status_locate_end` will be corrected to tab-indents